jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebInfConfiguration.java

```diff
@@ -315,42 +311,43 @@ public void unpack(WebAppContext context) throws IOException
             if (webApp.exists() && !webApp.isDirectory() && !webApp.toString().startsWith("jar:"))
             {
                 // No - then lets see if it can be turned into a jar URL.
-                Resource jarWebApp = JarResource.newJarResource(webApp);
-                if (jarWebApp.exists() && jarWebApp.isDirectory())
-                    webApp = jarWebApp;
+                // TODO
+//                Resource jarWebApp = JarResource.newJarResource(webApp);
+//                if (jarWebApp.exists() && jarWebApp.isDirectory())
+//                    webApp = jarWebApp;
```

Comment:
This is kind of fundamental!  We need this to be implemented.
I assume a filesystem will need to be added to the pool and then perhaps removed on stop.
Note you want to update to the latest to get the current working deployment code.